src/transforms/filter.jl

```diff
@@ -46,11 +64,33 @@ Drop all rows with missing values in table.
     DropMissing([col₁, col₂, ..., colₙ])
     DropMissing((col₁, col₂, ..., colₙ))
 
-Drop all rows with missing values in selects columns `col₁`, `col₂`, ..., `colₙ`.
+Drop all rows with missing values in selects columns `col₁`, `col₂`, ..., `colₙ`.  
+The `col` arguments must be the same type and the accepted types 
+for `col` arguments are: `Integer`, `Symbol` or `String`.
 
     DropMissing(regex)
 
 Drop all rows with missing values in columns that match with `regex`.
+
+# Examples
+```julia
+T = DropMissing()
+T = DropMissing("b", "c", "e")
+T = DropMissing([2, 3, 5])
+T = DropMissing((:b, :c, :e))
+T = DropMissing(r"[bce]]")
+```
+
+Note that columns affected by DropMissing will have their schema changed:
+```julia
+julia> table = (a = [1, missing, 2, 3], b = [missing, 1, 2, 3]);
+
+julia> T = DropMissing();
+
+julia> newtable, cache = apply(T, table);
+
+julia> newtable
+(a = [2, 3], b = [2, 3])
```

Comment:
```suggestion
```

Again this is a note, not something that end users should read right away. If you feel that this information is important, please create a ## Notes section at the bottom of the docstring with bullet points.